src/components/AccountSeed.js

```diff
@@ -132,6 +132,7 @@ export default class AccountSeed extends Component {
           style={[styles.input, invalidStyles]}
           multiline
           autoCapitalize="none"
+          textAlignVertical="top"
```

Comment:
Is this really needed? I think it looks better when the text is centred.

Reply:
Basic idea is to keep the design keep the same in both Android and iOS. Tried "alignment" to the middle, it will cause the scroll view to first move up, and move a little down after the first keystroke. I do not have a preference regarding the position, so I will just choose the one without issues then.